features/step_definitions/networking.rb

```diff
@@ -122,17 +122,16 @@
   teardown_add net_verify
 end
 
+
+CLUSTER_NETWORK = "clusternetwork"
```

Comment:
@rbbratta, it's not recommended to define a variable like this outside of the step.  You can probably just use the specification in line 134 `clusternetwork`

Reply:
is there way to define constants or symbols?

Reply:
@rbbratta why not just define that within your step?

Reply:
I refactored the clusternetwork subnet lookup into a single step, but we still have lots of duplicate strings around, I'm afraid of random typos and such every time I duplicate a long string.